### DISCUSSION TITLE
Sign out button is completely lowercase

### DISCUSSION THREAD

**Discussion by JacksonChen666** [ppy/osu#24026] [Q&A] [answered]
This seems inconsistent with all the other buttons nearby...

<img width="490" alt="osu!lazer logged in user dialog, highlighted button with the text 'sign out' in all lowercase" src="https://github.com/ppy/osu/assets/50755746/da57bf3b-5cd2-4d21-ae6f-ea0003575068">

**Comment by peppy** [comment]
Which should be an easy fix by using the correct string on osu-web, only it is title cased.

![2023-06-25 12 13 42@2x](https://github.com/ppy/osu/assets/191335/188cc405-053d-4f03-9f9c-93b59c28963b)

🤷‍♂️ can't win

**Reply by peppy** [reply, marked as answer]
https://github.com/ppy/osu/pull/24068
